### ISSUE TITLE
Disable desktop notification from Restyaboard

### ISSUE THREAD

**Issue by lmarikannan** [RestyaPlatform/board#1193]
From users,

Is there a way to stop or have control on notifications.. users are reporting seeing 100’s of them popping up on browser?

**Comment by lmarikannan** [comment]
FIxed on this commit - https://github.com/RestyaPlatform/board/commit/9877219b02927c8e373bfadf8b069f540e69331e, https://github.com/RestyaPlatform/board/commit/a9e2daa37f6ae0aae482b5164812d92494179e17

**Comment by rrjanbiah** [comment]
What are the changes done to fix this issue? Please describe. There are more changes

**Comment by lmarikannan** [comment]
These below changes done to fix this issue.

1. https://github.com/RestyaPlatform/board/commit/9877219b02927c8e373bfadf8b069f540e69331e#diff-0cc9662c3caf14bf344a5854ae79a106
2. https://github.com/RestyaPlatform/board/commit/9877219b02927c8e373bfadf8b069f540e69331e#diff-35df25b9af27bbb19c1176a9cdf9695d
3. https://github.com/RestyaPlatform/board/commit/9877219b02927c8e373bfadf8b069f540e69331e#diff-c60e449103faf0245fd84ac4277313fd
4. https://github.com/RestyaPlatform/board/commit/9877219b02927c8e373bfadf8b069f540e69331e#diff-d9c54f81517573dd0d620ca9bc0e2980

**Comment by rrjanbiah** [comment]
Bug title is not reflective of the commits. Please document properly about the other changes done

**Comment by lmarikannan** [comment]
We separated the desktop notifications to below cases
- List level notification - when updating color, card, move, archive, unarchive, delete
- Card level notification 1 - when updating color, due date, description, move, archive, unarchive, delete
- Card level notification 2 - when updating members
- Card level notification 3 - when updating labels
- Card level notification 4 - when updating checklist
- Card level notification 5 - when updating attachment

Desktop notifications are managed in two places. One is in admin settings page, and another one is in user profile page. So we integrated code in both templates and sync with browser notification state. And we controlled desktop notifications on footer view to check the activity type is in particular notification array. If it exists on that array, it'll show desktop notifications. And we can enable the browser notifications on user profile page.
